<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QLineEdit,QPlainTextEdit,QSpinBox,QDoubleSpinBox{

border:1px solid  #C0DCF2;
padding:2px;
background:none;
selection-background-color:#EEEEEE;
selection-color:#000000;
}
QLabel{
font-family:&quot;微软雅黑&quot;;
font-size:13px;

}
QLineEdit[echoMode=&quot;2&quot;]{
lineedit-password-character:9679;
}

QPushButton
{
    color: rgb(255, 255, 255);
    background-color: rgb(85, 170, 255);
    border:none;
    padding: 3px;
    font-family: &quot;微软雅黑&quot;;
    font-size: 12px;
    text-align: center;
}
QComboBox,QDateEdit,QSpinBox,QDoubleSpinBox{
border-radius:3px;
padding:3px 5px 3px 5px;
border:1px solid #C0DCF2;
}

QComboBox::drop-down,QDateEdit::drop-down{
subcontrol-origin:padding;
subcontrol-position:top right;
width:15px;
border-left-width:1px;
border-left-style:solid;
border-top-right-radius:3px;
border-bottom-right-radius:3px;
border-left-color:#C0DCF2;
}

QComboBox::down-arrow,QDateEdit::down-arrow{
image:url(:/image/array_down.png);
}
.QGroupBox{
border:1px solid #C0DCF2;
border-radius:5px;
font-family: &quot;微软雅黑&quot;;
}

QRadioButton{
color:#386487;
spacing:2px;
}

QRadioButton::indicator{
width:15px;
height:15px;
}

QRadioButton::indicator::unchecked{
image:url(&quot;D:/Documents/PyQt5/qt_protein/qt_protein/ui/images/radio_normal.png&quot;);
}

QRadioButton::indicator::checked{
image:url(&quot;D:/Documents/PyQt5/qt_protein/qt_protein/ui/images/radio_selected.png&quot;);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>421</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>fdr过滤</string>
    </property>
    <widget class="QPushButton" name="input_dir_btn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>182</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>输出目录</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_path">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>21</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>24</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>文件目录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="output_dir_btn">
     <property name="geometry">
      <rect>
       <x>227</x>
       <y>182</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>146</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>fdr</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>数据来源</string>
     </property>
     <widget class="QRadioButton" name="radiopfind">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> pfind</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radiomascot">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> mascot</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radiocoment">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> coment</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="output_path">
     <property name="geometry">
      <rect>
       <x>97</x>
       <y>182</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="fdr_lineedit">
     <property name="geometry">
      <rect>
       <x>97</x>
       <y>142</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.01</string>
     </property>
    </widget>
    <widget class="QPushButton" name="filter_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>开始过滤</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>421</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>合并多个过滤结果并去冗余</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>文件目录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="input_dir_mutibtn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>36</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_path_muti">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>37</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="output_path_muti">
     <property name="geometry">
      <rect>
       <x>89</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>输出目录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="output_dir_mutibtn">
     <property name="geometry">
      <rect>
       <x>219</x>
       <y>80</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
    <widget class="QPushButton" name="muti_btn">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>120</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>开始合并</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>421</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>单文件去冗余</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>源文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="input_dir_redundancybtn">
     <property name="geometry">
      <rect>
       <x>222</x>
       <y>36</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_path_redundancy">
     <property name="geometry">
      <rect>
       <x>91</x>
       <y>37</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="output_path_redundancy">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="output_dir_redundancybtn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>输出目录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="redundancy_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>127</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>467</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
